app/modules/onboarding/assets/js/pages/choose-features.js

```diff
@@ -0,0 +1,192 @@
+
+import Layout from '../components/layout/layout';
+import PageContentLayout from '../components/layout/page-content-layout';
+
+export default function ChooseFeatures() {
+	const { state, updateState, getStateObjectToUpdate } = useContext( OnboardingContext ),
+		{ ajaxState, setAjax } = useAjax(),
+		[ noticeState, setNoticeState ] = useState( null ),
+		[ siteNameInputValue, setSiteNameInputValue ] = useState( state.siteName ),
+		[ featureWasSelected, setFeatureWasSelected ] = useState( true ),
+		selectedFeatures = [],
+		tierName = __( 'Essential', 'elementor' ),
+		pageId = 'chooseFeatures',
+		nextStep = 'goodToGo',
+		navigate = useNavigate(),
+		nameInputRef = useRef(),
+		options = [
+			{
+				plan: 'essential',
+				text: __( 'Templates & Theme Builder', 'elementor' ),
+			},
+			{
+				plan: 'advanced',
+				text: __( 'WooCommerce Builder', 'elementor' ),
+			},
+			{
+				plan: 'essential',
+				text: __( 'Lead Collection & Form Builder', 'elementor' ),
+			},
+			{
+				plan: 'essential',
+				text: __( 'Dynamic Content', 'elementor' ),
+			},
+			{
+				plan: 'advanced',
+				text: __( 'Popup Builder', 'elementor' ),
+			},
+			{
+				plan: 'advanced',
+				text: __( 'Custom Code & CSS', 'elementor' ),
+			},
+			{
+				plan: 'essential',
+				text: __( 'Motion Effects & Animations', 'elementor' ),
+			},
+			{
+				plan: 'advanced',
+				text: __( 'Notes & Collaboration', 'elementor' ),
+			},
+		],
+		actionButton = {
+			text: __( 'Upgrade Now', 'elementor' ),
+			onClick: () => {
+				elementorCommon.events.dispatchEvent( {
+					event: 'next',
+					version: '',
+					details: {
+						placement: elementorAppConfig.onboarding.eventPlacement,
+						step: state.currentStep,
+					},
+				} );
+
+				// Only run the site name update AJAX if the new name is different than the existing one and it isn't empty.
+				if ( nameInputRef.current.value !== state.siteName && '' !== nameInputRef.current.value ) {
+					setAjax( {
+						data: {
+							action: 'elementor_update_site_name',
+							data: JSON.stringify( {
+								siteName: nameInputRef.current.value,
+							} ),
+						},
+					} );
+				} else if ( nameInputRef.current.value === state.siteName ) {
+					const stateToUpdate = getStateObjectToUpdate( state, 'steps', pageId, 'completed' );
+
+					updateState( stateToUpdate );
+
+					navigate( 'onboarding/' + nextStep );
+				} else {
+					const stateToUpdate = getStateObjectToUpdate( state, 'steps', pageId, 'skipped' );
+
+					updateState( stateToUpdate );
+
+					navigate( 'onboarding/' + nextStep );
+				}
+			},
+		};
+
+	let skipButton;
+
+	if ( 'completed' !== state.steps[ pageId ] ) {
+		skipButton = {
+			text: __( 'Skip', 'elementor' ),
+		};
+	}
+
+	if ( ! siteNameInputValue || 0 === selectedFeatures.length ) {
+		actionButton.className = 'e-onboarding__button--disabled';
+	}
+
+	// Run the callback for the site name update AJAX request.
+	useEffect( () => {
```

Comment:
I see this is copied from `site-name.js`. I assume we will remove it from there once the experiment is merged. Or do we need it in both places?

Reply:
No, it will all be removed once we will delete the experiment.